integration-tests/http/__tests__/api-key/admin/publishable-key.spec.ts

```diff
@@ -0,0 +1,291 @@
+import { medusaIntegrationTestRunner } from "medusa-test-utils"
+import {
+  adminHeaders,
+  createAdminUser,
+} from "../../../../helpers/create-admin-user"
+
+jest.setTimeout(50000)
+
+medusaIntegrationTestRunner({
+  env: {},
+  testSuite: ({ dbConnection, getContainer, api }) => {
+    describe("Publishable Keys - Admin", () => {
+      let pubKey1
+      let pubKey2
+
+      beforeEach(async () => {
+        await createAdminUser(dbConnection, adminHeaders, getContainer())
+
+        // BREAKING: Before the ID of the token was used in request headers, now there is a separate `token` field that should be used
+        pubKey1 = (
+          await api.post(
+            "/admin/api-keys",
+            { title: "sample key", type: "publishable" },
+            adminHeaders
+          )
+        ).data.api_key
+        pubKey2 = (
+          await api.post(
+            "/admin/api-keys",
+            // BREAKING: The type field is now required
+            { title: "just a title", type: "publishable" },
+            adminHeaders
+          )
+        ).data.api_key
+      })
+
+      // BREAKING: The URL changed from /admin/api-keys to /admin/api-keys, as well as the response field
```

Comment:
whats the difference in the url here?

Reply:
Ah god damn autocomplete 😄 It was `/admin/publishable-api-keys`